frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
+import * as React from 'react';
+import { VictoryStack, VictoryBar } from 'victory';
+import { Tooltip } from '@console/internal/components/utils/tooltip';
+import { getTaskStatus, PipelineRun, mapColorMessage } from '../../utils/pipeline-augment';
+import TaskStatusToolTip from './TaskStatusToolTip';
+import './PipelineTaskStatus.scss';
+
+interface PipelineTaskStatusProps {
+  pipelinerun: PipelineRun;
+}
+
+export const PipelineTaskStatus: React.FC<PipelineTaskStatusProps> = ({ pipelinerun }) => {
+  const taskStatus = getTaskStatus(pipelinerun);
+  return (
+    <div>
+      <Tooltip content={[<TaskStatusToolTip key="tooltip" taskStatus={taskStatus} />]}>
+        <VictoryStack horizontal height={32} padding={0} domainPadding={0}>
+          {Object.keys(mapColorMessage).map((bar) => {
+            return (
+              taskStatus[bar] > 0 && (
+                <VictoryBar
+                  key={bar}
+                  data={[{ x: 2, y: taskStatus[bar] }]}
+                  barRatio={0.9}
+                  barWidth={32}
+                  padding={{ right: 15 }}
```

Comment:
Where is this being applied? Looks the same without this value.

Reply:
removing this